### ISSUE TITLE
Always uses default search query (0)

### ISSUE THREAD

**Issue by nloveladyallen** [hwclass/node-cli-google#15]
No matter what is passed as an argument, it returns search results for [0](https://www.google.com/search?q=0).

```
$ search foo bar
 0:  https://twitter.com/0?lang=en
 1:  https://en.wikipedia.org/wiki/0
 2:  https://en.wikipedia.org/wiki/0_(number)
 3:  https://www.ixl.com/math/grade-3/multiply-by-0
 4:  https://www.theguardian.com/football/live/2016/jun/05/sweden-v-wales-pre-euro-2016-international-friendly-live
 5:  http://www.bbc.com/sport/football/36407575
 6:  http://mathforum.org/dr.math/faq/faq.0.to.0.power.html
 7:  https://play.google.com/store/apps/details?id=com.clearskyapps.fitnessfamily.PushupsPro&hl=en
 8:  https://www.youtube.com/user/Apple
prompt: Update selection (blank to accept)>>:
```

**Comment by nloveladyallen** [comment]
On investigation, the issue has been fixed in 1.2.3, but package.json is hard-coded to install version 1.2.2. 1.2.2 should be removed from the repository and 1.2.3 should be renamed just "node-cli-google.js." package.json should also be updated accordingly.

**Comment by hwclass** [comment]
hi @nloveladyallen,

thanks for the contribution. will be checking the bug.

Take care.
